include/SDL3/SDL_pen.h

```diff
@@ -107,7 +107,13 @@ typedef enum SDL_PenAxis
 #define SDL_PEN_TIP_INK    SDL_PEN_FLAG_INK_BIT_INDEX     /**< Regular pen tip (for drawing) touched the surface */
 #define SDL_PEN_TIP_ERASER SDL_PEN_FLAG_ERASER_BIT_INDEX  /**< Eraser pen tip touched the surface */
 
-/* Pen capabilities reported by SDL_GetPenCapabilities */
+/**
+ * Pen capabilities reported by SDL_GetPenCapabilities.
+ *
+ * \since This datatype is available since SDL 3.0.0.
+ */
+typedef Uint64 SDL_PenCapabilityFlags;
```

Comment:
Uint32? Or did you mean to change it?

Reply:
Whoops, left that in to check for compiler warnings, good catch!

Reply:
Fixed in 1936e91d85dcf4087ac82769aa27cf3f7630dcb4.